shellcheck-all.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Do you we need this file outside of github actions? If not, the content could easily go into the actions file, which supports shell commands too.

Reply:
Probably, that will allow the PR submitter to run this script on their dev machine before having to push to see the results. That way they don't have to dig into the yml code to see what they have to do in order to pass shellcheck.

Reply:
That depends. If you're just calling shell-check for a single file, then the script doesn't bring any additional value. They will need to install shell-check anyways (or at least use the online version).